.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Chromatic'
+
+on: push
+
+env:
+  NODE_VERSION: 16
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
+      - uses: actions/setup-node@v3
+      - name: Load cached node_modules
+        uses: actions/cache@v3
+        run: yarn
+        with:
```

Comment:
actions cache doesnt need node_version

Reply:
its for yarn you mentioned

Reply:
i meant yarn needs 
- uses: actions/setup-node@v3
   with:
       node-version: ${{ env.NODE_VERSION }}